packages/grafana-data/src/transformations/transformers/histogram.ts

```diff
@@ -220,7 +220,7 @@ export function buildHistogram(frames: DataFrame[], options?: HistogramTransform
           ...field,
           config: {
             ...field.config,
-            unit: undefined,
+            unit: field.config.unit === 'short' ? 'short' : undefined,
```

Comment:
this adds an allowance for `short`, which is still unitless, but could be useful for large Y counts